data-migration/migrate-sharding-mysql-tidb-less-tb.md

```diff
@@ -0,0 +1,239 @@
+
+In this example, `sale_01` and `sale_02` have the same table structure as follows
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE `sale_01` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `sid` bigint(20) NOT NULL,
+  `pid` bigint(20) NOT NULL,
+  `comment` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `sid` (`sid`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1
+```
+
+The `id` column is the primary key, and the `sid` column is the sharding key. The `id` column is auto-incremental, and duplicated multiple sharded table ranges will cause data conflicts. The `sid` can ensure that the index is globally unique, so you can follow the steps in [Remove the primary key attribute of the auto-incremental primary key](https://docs.pingcap.com/tidb-data-migration/stable/shard-merge-best-practices#remove-the-primary-key-attribute-from-the-column) to bypasses the `id` column.
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE `sale` (
+  `id` bigint(20) NOT NULL,
+  `sid` bigint(20) NOT NULL,
+  `pid` bigint(20) NOT NULL,
+  `comment` varchar(255) DEFAULT NULL,
+  INDEX (`id`),
+  UNIQUE KEY `sid` (`sid`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1
+```
+
+## Step 1: Load data sources
+
+Create a file named `source1.yaml`. The configuration file is as follows:
+
+{{< copyable "shell-regular" >}}
+
+```yaml
+# Configuration.
+source-id: "mysql-01" # Must be unique.
+
+# Specifies whether DM-worker pulls binlogs with GTID (Global Transaction Identifier).
+# The prerequisite is that you have already enabled GTID in the upstream MySQL.
+# If the upstream database has enabled automatic switch of primary and standby, you must use the GTID mode.
+enable-gtid: false
+from:
+  host: "${host}" # For example: 172.16.10.81
+  user: "root"
+  password: "${password}" # Plaintext passwords are supported but not recommended. It is recommended that you use dmctl encrypt to encrypt plaintext passwords.
+  port: 3306
+```
+
+Run the following command in a terminal. Use `tiup dmctl` to load the data source configuration into the DM cluster:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup dmctl --master-addr ${advertise-addr} operate-source create source1.yaml
+```
+
+The parameters are described as follows.
+
+|Parameter      | Description |
+|-              |-            |
+|--master-addr         | {advertise-addr} of any DM-master node in the cluster that dmctl connects to. For example: 172.16.10.71:8261|
+|operate-source create | Load data sources to the DM cluster. |
+
+Repeat the above steps until all data sources are added to the DM cluster.
+
+## Step 2: Configure the migration task
+
+Create a file named `task1.yaml` and writes the following content to it:
+
+{{< copyable "shell-regular" >}}
+
+```yaml
```

Comment:
The common comments in the config here should better be consistent with all the documents referring a task config file.